### PR TITLE
export `setSpecialConfigValues` to window to allow to creation of

### PR DESCRIPTION
**Pull request by goapunk**
a CKEDITOR instance from js (e.g. used in the documents app)

### CODE REVIEW

File: django_ckeditor_5/static/django_ckeditor_5/app.js [hunk 1]
```diff
@@ -19,6 +19,72 @@ function getCookie(name) {
...
+    if(config.mediaEmbed) {
+        config.mediaEmbed.providers = [
+            {
+                name: 'youtube',
+                url: [
+                        /^(?:m\.)?youtube\.com\/watch\?v=([\w-]+)(?:&t=(\d+))?/,
+                        /^(?:m\.)?youtube\.com\/v\/([\w-]+)(?:\?t=(\d+))?/,
+                        /^youtube\.com\/embed\/([\w-]+)(?:\?start=(\d+))?/,
+                        /^youtu\.be\/([\w-]+)(?:\?t=(\d+))?/
+                ],
```
Comment (philli-m) [inline django_ckeditor_5/static/django_ckeditor_5/app.js:45]: will this work with youtube-nocookie urls? its what you get if you use the embed url and click the box 'enhaced privacy` checkbox

**Review by philli-m — CHANGES_REQUESTED** [review]
don't think it will work with no cookie but can't test

File: django_ckeditor_5/static/django_ckeditor_5/app.js (hunk 1 above)
Comment (philli-m) [reply django_ckeditor_5/static/django_ckeditor_5/app.js:45]: also should we wrap all options with maybe mobile check or is that not required for other urls?

**Comment by goapunk** [comment]
> don't think it will work with no cookie but can't test

you can test on dev, it's the same logic 
edit: added test instruction in mb pr

File: django_ckeditor_5/static/django_ckeditor_5/app.js (hunk 1 above)
Comment (goapunk) [reply django_ckeditor_5/static/django_ckeditor_5/app.js:45]: I copied this from ckeditor, I didn't look too closely. I think `youtou.be` just redirects to `youtube.com` so it should be ok

Comment (goapunk) [reply django_ckeditor_5/static/django_ckeditor_5/app.js:45]: but yeah most likely won't work with youtube-nocookie

Comment (goapunk) [reply django_ckeditor_5/static/django_ckeditor_5/app.js:45]: added a fix

**Review by philli-m — APPROVED** [review]
Nice videos work as does ckeditor in docs

**Pull request merged**